### PR TITLE
Add python37 DU v2

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: language_version_metadata/Python37_DU__2__metadata.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[{
+        "version": 2,
+        "imageType": "SERVING",
+        "processor": "CPU",
+        "mlPackageLanguage": "PYTHON37_DU",
+        "languageGroup": "DU",
+        "baseImage": "uipathmlcontainerregistry.azurecr.io/aifabric_base_image:NNN-AIF",
```
Comment (vivekjhauipath) [inline language_version_metadata/Python37_DU__2__metadata.json:7]: replace NNN with 21.4.0

**Review by vivekjhauipath — CHANGES_REQUESTED** [review]
image tag should be corrected

**Review by vivekjhauipath — APPROVED** [review]
LGTM

**Comment by irinel-birsan-uipath** [comment]
image

**Review by adeepsri — APPROVED** [review]
LGTM

**Comment by mohitseth-nitrr** [comment]
@irinel-birsan-uipath Can you attach the ticket here ? I want to understand what are these changes for and why do they belong to separate folder on ai-customer-scripts instead of metadata folder.

**Comment by mohitseth-nitrr** [comment]
> 
> 
> @irinel-birsan-uipath Can you attach the ticket here ? I want to understand what are these changes for and why do they belong to separate folder on ai-customer-scripts instead of metadata folder.

confirmed with @adeepsri @vivekjhauipath  ... looks like this is something new

**Pull request merged**